src/index.ts

```diff
@@ -136,112 +158,131 @@ const MATCH_SASS_FILENAME_RE = /\.sass$/,
+        .then((res: SassRenderResult) =>
+          processRenderResponse(
+            pluginOptions,
+            filePath,
+            pluginState,
+            res.css.toString().trim(),
+          ).then((result) => [res, result]),
         )
-        .then(([res, codeResult]: [SassRenderResult, RollupPluginSassProcessorFnOutput]) => {
+        .then(
+          ([res, codeResult]: [
+            SassRenderResult,
+            RollupPluginSassProcessorFnOutput,
+          ]) => {
             // @todo Do we need to filter this call so it only occurs when rollup is in 'watch' mode?
             res.stats.includedFiles.forEach((filePath: string) => {
               this.addWatchFile(filePath);
             });
 
             return {
-                code: codeResult || '',
-                map: {mappings: res.map ? res.map.toString() : ''}
+              code: codeResult || "",
+              map: { mappings: res.map ? res.map.toString() : "" },
             };
-        }); // @note do not `catch` here - let error propagate to rollup level.
+          },
+        ); // @note do not `catch` here - let error propagate to rollup level.
     },
 
-    generateBundle(generateOptions: { file?: string },
-                   bundle: { [fileName: string]: RollupAssetInfo | RollupChunkInfo },
-                   isWrite: boolean
+    generateBundle(
+      generateOptions: { file?: string },
+      bundle: { [fileName: string]: RollupAssetInfo | RollupChunkInfo },
+      isWrite: boolean,
     ): Promise<any> {
-      if (!isWrite || (!pluginOptions.insert && (!pluginState.styles.length || pluginOptions.output === false))) {
+      if (
+        !isWrite ||
+        (!pluginOptions.insert &&
+          (!pluginState.styles.length || pluginOptions.output === false))
+      ) {
         return Promise.resolve();
       }
 
-      const {styles} = pluginState,
-        css = styles.map(style => style.content).join(''),
-        {output, insert} = pluginOptions;
+      const { styles } = pluginState;
+      const css = styles.map((style) => style.content).join("");
+      const { output, insert } = pluginOptions;
 
-      if (typeof output === 'string') {
-        return fs.promises.mkdir(dirname(output as string), {recursive: true})
+      if (typeof output === "string") {
+        return fs.promises
+          .mkdir(dirname(output as string), { recursive: true })
           .then(() => fs.promises.writeFile(output as string, css));
-      } else if (typeof output === 'function') {
-        return Promise.resolve((output as RollupPluginSassOutputFn)(css, styles));
+      } else if (typeof output === "function") {
+        return Promise.resolve(
+          (output as RollupPluginSassOutputFn)(css, styles),
+        );
       } else if (!insert && generateOptions.file && output === true) {
         let dest = generateOptions.file;
 
-        if (dest.endsWith('.js') || dest.endsWith('.ts')) {
+        if (dest.endsWith(".js") || dest.endsWith(".ts")) {
```

Comment:
Could we change the prettier rule to favor single-quotes instead of double ones?

Reply:
add `.prettierrc.json` with `"singleQuote": true`